src/client.rs

```diff
@@ -4,7 +4,7 @@
 // This file may not be copied, modified, or distributed except according to those terms.
 
 use {Reactor, WireError};
-use bincode::serde::DeserializeError;
+use bincode::Error as BincodeError;
```

Comment:
Nit: `use bincode;` then refer to `bincode::Error` to be consistent with other modules.